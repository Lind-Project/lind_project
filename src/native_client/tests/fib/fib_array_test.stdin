rpc fib i(1) i(1) * I(10)
rpc fib i(1) i(2) * I(15)
