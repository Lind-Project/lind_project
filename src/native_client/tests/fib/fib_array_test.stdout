rpc call initiated fib:ii:I
input 0:  i(1)
input 1:  i(1)
rpc call complete fib:ii:I
output 0:  I(10,1,1,2,3,5,8,13,21,34,55)

rpc call initiated fib:ii:I
input 0:  i(1)
input 1:  i(2)
rpc call complete fib:ii:I
output 0:  I(15,1,2,3,5,8,13,21,34,55,89,144,233,377,610,987)

